Experimental/MultiplayerUseCases/Assets/Common/Scripts/NetworkManagerUI.cs

```diff
@@ -86,20 +86,11 @@ void StartHost(ClickEvent evt)
 
         void StartClient(ClickEvent evt)
         {
-            SetNetworkPortAndAddress(k_DefaultPort, m_AddressInputField.value, k_DefaultServerListenAddress);
```

Comment:
I think this was removed by mistake, it shoudl be re-added, otherwise the client is not going to know who to connect to